src/openrct2/ride/RideData.h

```diff
@@ -324,6 +324,103 @@ struct TrackDrawerDescriptor
     bool HasCoveredPieces() const;
 };
 
+enum class RtdFlag : uint8_t
+{
+    hasTrackColourMain = 0,
+    hasTrackColourAdditional = 1,
+    hasTrackColourSupports = 2,
+    // Set by flat rides, tower rides and shops/stalls.
+    hasSinglePieceStation = 3,
+    hasLeaveWhenAnotherVehicleArrivesAtStation = 4,
+    canSynchroniseWithAdjacentStations = 5,
+    // Used only by boat Hire and submarine ride
+    trackMustBeOnWater = 6,
+    hasGForces = 7,
+    // Used by rides that can't have gaps, like those with a vertical tower,
+    // such as the observation tower.
+    cannotHaveGaps = 8,
+    hasDataLogging = 9,
+    hasDrops = 10,
+    noTestMode = 11,
+    // Set on rides with two varieties, like the u and o shapes of the dinghy slide
+    // and the dry and submerged track of the water coaster.
+    hasCoveredPieces = 12,
+    // Used only by maze, spiral slide and shops
```

Comment:
This is _much_ better, thanks. I think it would be even more self-explanatory if we add an extra newline before a comment, though. Just so no-one gets the idea that the comments apply to the preceding line.